testsuite/integration-arquillian/tests/base/testsuites/database-suite.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash -e
 
 DIR=`readlink -f $0 | xargs dirname`
-cd $DIR
+cd "$DIR"
 
 TESTSUITE_FILE="database-suite"
 
 TESTS=`cat $TESTSUITE_FILE`
 
-echo $TESTS | sed 's/ /,/g'
\ No newline at end of file
+echo "${TESTS// /,}"
```

Comment:
This is a fix. See https://www.shellcheck.net/wiki/SC2001

Reply:
@stianst @hmlnarik I'm wondering if we should consider using https://github.com/koalaman/shellcheck/wiki in our `kc.sh` script.

@hmlnarik I appreciate it if you could also spend some time looking at `kc.sh` and see what we can improve there. Wdyt?